posthog/utils.py

```diff
@@ -859,7 +859,7 @@ def get_instance_available_sso_providers() -> Dict[str, bool]:
         "SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET",
         None,
     ):
-        if bypass_license or (license is not None and AvailableFeature.GOOGLE_LOGIN in license.available_features):
+        if bypass_license or (license is not None and AvailableFeature.SOCIAL_SSO in license.available_features):
```

Comment:
This should only impact licensed self-hosted instances that aren't our cloud. I don't think we have any more of those? cc @simfish85